SCRAM/GMake/Makefile.rules

```diff
@@ -646,8 +646,8 @@ endef
 define link_cuda_objs
   @$(startlog_$(1)) [ -d $(@D) ] ||  $(CMD_mkdir) -p $(@D) &&\
   $(CMD_echo) ">> Cuda Device Link $@ " &&\
-  $(VERB_ECHO) $(SCRAM_PREFIX_COMPILER_COMMAND) nvcc -dlink $(call AdjustFlags,$1,,CPPFLAGS) --compiler-options "$(CUDA_CFLAGS)" $(CUDA_FLAGS) $? -o $@ &&\
-              ($(SCRAM_PREFIX_COMPILER_COMMAND) nvcc -dlink $(call AdjustFlags,$1,,CPPFLAGS) --compiler-options "$(CUDA_CFLAGS)" $(CUDA_FLAGS) $? -o $@) $(endlog_$(1))
+  $(VERB_ECHO) $(SCRAM_PREFIX_COMPILER_COMMAND) $(NVCC) -dlink $(call AdjustFlags,$1,,CPPFLAGS) --compiler-options "$(CUDA_CFLAGS)" $(CUDA_FLAGS) $? -o $@ &&\
+              ($(SCRAM_PREFIX_COMPILER_COMMAND) $(NVCC) -dlink $(call AdjustFlags,$1,,CPPFLAGS) --compiler-options "$(CUDA_CFLAGS)" $(CUDA_FLAGS) $? -o $@) $(endlog_$(1))
```

Comment:
I don't know why, but this lets `nvcc` pick up the correct setup

Reply:
thanks, I should have used $(NVCC) in the first place.